internal/learn/the-community/qanda.md

```diff
@@ -0,0 +1,98 @@
+_Source: https://twitter.com/ercwl/status/1527071088676577282_
+
+## A:
+Look at the state of blockchain ATM. Most now aim to scale through L2 bc L1 just sucks for most. In addition they reduce the number of validators (Solana is extreme here, but you see it also at Eth2.0, BSC etc.), to achieve speed. (Fewer validators can form a consensus quicker.)
+Also a blockchain is a linear list of tx in a block. If one conflicts you have to reroll the good ones as well. IOTA essentially says: you don't need a serial list, but 1) we see a way how tx can be executed  in parallel ("blocksize=1tx") and avoid serial dependency, and 2) we extend Nakamoto consensus to have thousands/millions of validators in parallel (the initial spirit of blockchain) without compromising security.  
+All of this with subsecond finality, security and high scalability as "blocksize=1" can then be executed on multiple CPU cores (not possible with blocks) to scale TPS and thus take "blockchain" to the next level.
+Next level meaning:
+- no uncalculatable tx fees (fee=0)
+- super green, as there are no miners etc.
+- very robust & scalable infrastructure
+- all with EVM (more down the road).
+
+The 0 fee also enables 1000s of SC chains to easy exchange L1 assets (tokens, NFT's...) for free across chains w/o hackable bridges. 
+Fundamentally(!) change how we see DLT and how it can scale, to finally mitigate the limits of blockchain and serve real use-cases/requirements of industry/crypto/web3 better than anything built on blockchain.
+
+_Source: https://threadreaderapp.com/thread/1527201480574480385.html_
+
+# Q: Why use IOTA? No one uses it. Why not an active chain like `insert blockchains here`?
+
+# Q: Since there are no gas fees, what’s the point of iota?
+## A:
+Past coordicide you will need Mana to participate in the network. Mana is generated by holding IOTA. Nodes that do not hold IOTA, but still want to participate in the IOTA network or simply have a very high throughput demand can buy access mana via some market or you could also possibly pledge your mana to a node in return for rewards.
+https://blog.iota.org/explaining-mana-in-iota-part-2
+And while transactions are "free", smart contracts can demand a fee for their execution.
+
+_Source: https://www.reddit.com/r/Iota/comments/us4sc1/comment/i91vb5w_
+
+## A:
+
+Other applications that require the IOTA token:
+
+**Digital assets**
+IOTA is a multi-asset ledger. Most DLTs fall into the single-asset ledger category as they are only able to track ownership of a specific base currency within their ledger. Multi-asset ledgers, on the other hand, are able to manage multiple native tokens in the same ledger as the base currency. Since IOTA’s base ledger allows for feeless transactions, IOTAs multi-asset ledger is able to perform feeless transfers of any native tokens, which is a unique innovation in the field.
```

Comment:
```suggestion
IOTA is a multi-asset ledger. Most DLTs fall into the single-asset ledger category as they can only track ownership of a specific base currency within their ledger. On the other hand, multi-asset ledgers can manage multiple native tokens in the same ledger as the base currency. Since IOTA’s base ledger allows for feeless transactions, IOTA's multi-asset ledger can perform feeless transfers of native tokens, a unique innovation in the field.
```